tslint.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "tslint-config-standard"
```

Comment:
My company is `@uncovertruth/tslint-config`  host myself, extends standard linter